### ISSUE TITLE
Remove 'variables' section from inventories/ api endpoint

### ISSUE THREAD

**Issue by gamuniz** [ansible/awx#4840]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY

If using the UI to manage top level inventory variables having a large list of variables can cause performance issues across the network. In some cases the calls are generating 7.2 MB worth of data on every job launch prompt. 

This data is already provided in the related section and is extraneous on the inventories/ endpoint

**Comment by ryanpetrello** [comment]
Only downside is that this represents a backwards-incompatible API change.

@gamuniz are you saying we should remove this from the _list_ view?  We're probably not likely to remove it from the detail view.

**Comment by wenottingham** [comment]
I would note that we include extra_vars on the list view for both jobs and job templates, which is a similar source of unbounded data.

**Comment by gamuniz** [comment]
@ryanpetrello Yeah the list view is what we're currently using to populate the launch on prompt for inventories, having multiple job templates each prompting for inventories can cause several calls to inventories/ which loads all data as @wenottingham mentioned currently its unbounded data. If removing the variables causes incompatible api changes, perhaps changing the query used in prompts would work.
